src/pymodaq/utils/leco/pymodaq_listener.py

```diff
@@ -234,42 +269,51 @@ def queue_command(self, command: ThreadCommand) -> None:
                 param_dict_str=ioxml.parameter_to_xml_string(param).decode())
 
         elif command.command == LECOMoveCommands.POSITION:
-            value = command.attribute[0]  # type: ignore
+            value = command.attribute
+            if isinstance(value, (list, tuple)):
+                value = value[0]  # for backward compatibility with attributes list
             self.communicator.ask_rpc(receiver=self.remote_name,
                                       method="set_position",
-                                      **binary_serialization_to_kwargs(value, data_key="position"),
+                                      **binary_serialization_to_kwargs(pymodaq_object=value, data_key="position"),
                                       )
 
         elif command.command == LECOMoveCommands.MOVE_DONE:
-            value = command.attribute[0]  # type: ignore
+            value = command.attribute
+            if isinstance(value, (list, tuple)):
+                value = value[0]  # for backward compatibility with attributes list
             self.communicator.ask_rpc(receiver=self.remote_name,
                                       method="set_move_done",
                                       **binary_serialization_to_kwargs(value, data_key="position"),
                                       )
 
         elif command.command == 'x_axis':
```

Comment:
this is deprecated as the axes are now included in the data object.

Reply:
I looked at the Move module and got inspired from there.

Reply:
yeah I should have removed this long time ago... I'm opening an issue